frontend/src/scenes/pipeline/hogfunctions/hogFunctionConfigurationLogic.tsx

```diff
@@ -891,7 +899,7 @@ export const hogFunctionConfigurationLogic = kea<hogFunctionConfigurationLogicTy
             }
             const { id, name, type } = values.hogFunction
             await deleteWithUndo({
-                endpoint: `projects/${teamLogic.values.currentTeamId}/hog_functions`,
+                endpoint: `projects/${values.currentProjectId}/hog_functions`,
```

Comment:
Could probably get away with `values.currentProject.id` here and remove the need for the `currentProjectId` value